app/views/posts/index.html.erb

```diff
@@ -1,30 +1,47 @@
 <div class="user_c">
+
     <div class="user_data">
-        <div class="user_img">
-      <%= image_tag("Tom.png", width:'150px', height:'170px') %>
-    </div>
+
       <h2><%= @user.name%></h2>
+
       <p>Number of posts: <%= @user.posts_counter %></p>
+
     </div>
```

Comment:
- [ ] Also with the number of posts. I have added only two posts as you can see below screenshot but it's showing 4. Please fix that.

![Screenshot from 2022-02-25 02-03-24](https://user-images.githubusercontent.com/22434660/155603207-e7bf9ee3-67cf-4bc3-a1e6-57d7403eeaaf.png)